angular-segmentio.js

```diff
@@ -56,6 +49,23 @@ angular.module('segmentio', ['ng'])
                 firstScript.parentNode.insertBefore(script, firstScript);
             };
 
+          // Method to target and remove the analytics script element from the DOM
+          // which allows SegmentIO and Pendo to be toggled off at logout.
+          service.unload = function() {
+              var segmentScript = document.getElementById('segmentio');
+              if (segmentScript) {
+                  segmentScript.parentNode.removeChild(segmentScript);
+              }
+          };
+
+          // Remove any existing Pendo elements in the DOM
+          service.removePendoCTA = function() {
```

Comment:
I guess there is no harm here, but I'm not clear on what is actually calling this function.  I assume unload will be triggered here by a normal document.unload?  Or is there code going in elsewhere that will call these functions if they exist?

Reply:
@jastorquia yup these methods are being added to the service which is being injected into the topnavbar.js in the monolith which calls these additional service functions when the user logs out https://github.com/BetterWorks/BetterWorks/pull/21596/files